varnishgather

```diff
@@ -102,6 +102,9 @@ banner()
 run()
 {
 	incr
+
+	which "$1" >/dev/null 2>&1 || return 0
```

Comment:
@tfheen you are my hero :D

@fgsch would it work for you? I have successfully tested it in bash, csh, dash, ksh, tcsh, zsh and somewhat successfully in fish (the latter resolves built-ins after executables in the path)

However in my latest patches I no longer use `type` as per your instructions.